pkg/gofr/container/container.go

```diff
@@ -39,6 +40,21 @@ type Container struct {
 	Mongo datasource.Mongo
 }
 
+func (c *Container) Close() (err error) {
+	if c.PubSub != nil {
+		err = errors.Join(c.PubSub.Close())
+	}
+
+	if c.SQL != nil {
+		sqlDB, ok := c.SQL.(*sql.DB)
+		if ok && sqlDB != nil {
+			err = errors.Join(sqlDB.Close())
```

Comment:
Joining one error sounds odd

Either you wanted to join with another error, or you could use err =